<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationDlg</class>
 <widget class="QDialog" name="StationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор пункта</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="addBtn">
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>218</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="StationWidget" name="stationWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="isMapButtonVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isHorisontal" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>StationDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
